.github/workflows/format.yml

```diff
@@ -0,0 +1,26 @@
+name: Linting
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+
+jobs:
+  lint:
+    name: Linting
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout Code
+        uses: actions/checkout@v2
+      - name: Update packages
+        run: sudo apt-get update
+      - name: Set up Python 3.9
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.9
+      - name: Install Dependencies
+        run: pip install -e '.[dev]'
+      - name: Run formatting check
+        run: make format
```

Comment:
why both this and linting?

Reply:
i can put them in the same workflos if you like